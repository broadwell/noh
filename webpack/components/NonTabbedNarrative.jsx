import React from "react";
import PropTypes from "prop-types";
import { Markup } from "interweave";

const NonTabbedNarrative = props => <Markup content={props.narrative} />;

NonTabbedNarrative.propTypes = {
  narrative: PropTypes.string
};

NonTabbedNarrative.defaultProps = {
  narrative: ""
};

export default NonTabbedNarrative;
