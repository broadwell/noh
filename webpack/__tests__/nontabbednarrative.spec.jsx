import React from "react";
import { shallow } from "enzyme";
import NonTabbedNarrative from "../components/NonTabbedNarrative";

describe("<NonTabbedNarrative>", () => {
  const nonTabbedAnalysis =
    "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.";

  it("renders as expected", () => {
    const component = shallow(
      <NonTabbedNarrative narrative={nonTabbedAnalysis} />
    );
    expect(component).toMatchSnapshot();
  });
});
