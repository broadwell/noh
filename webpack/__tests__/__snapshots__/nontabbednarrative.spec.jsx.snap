// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NonTabbedNarrative> renders as expected 1`] = `
<Markup
  content="Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam."
  emptyContent={null}
  parsedContent={null}
  tagName="div"
/>
`;
